learning_journal/views/default.py

```diff
@@ -0,0 +1,93 @@
+from pyramid.response import Response
+from pyramid.view import view_config
+
+from sqlalchemy.exc import DBAPIError
+
+from ..models import Journal
+
+
+# @view_config(route_name='default', renderer='../templates/mytemplate.jinja2')
+# def my_view(request):
+#     try:
+#         query = request.dbsession.query(Journal)
+#         journal = query.filter(Journal.title == 'Test').first()
+#     except DBAPIError:
+#         return Response(db_err_msg, content_type='text/plain', status=500)
+#     return {'title': journal.title, 'project': 'learning_journal'}
+
+
+@view_config(
+    route_name='home', renderer='learning_journal:templates/home.html'
+)
+def home_view(request):
+    """Render home.html at '/'"""
+    if request.method == 'POST':
+        title = request.POST['title']
+        body = request.POST['body']
+        new_entry = Journal(title=title, body=body)
+        request.dbsession.add(new_entry)
+    try:
+        journal_entries = request.dbsession.query(Journal).all()
+    except DBAPIError:
+        return Response(db_err_msg, content_type='text/plain', status=500)
```

Comment:
Here, I would allow any `DBAPIError` to go unhandled.  The framework will take care of converting it to a 500 response in production, and in development you'll see a helpful traceback.